### PR TITLE
Prevent some UI elements from wrapping when the window is resized

### PR DESCRIPTION
**Pull request by peterp**
1. The menu
2. Breadcrumbs and title.
3. Sidebar actions and page actions.

### CODE REVIEW

**Comment by justinvdm** [comment]
Other than my comment, I'm :+1: , although my css (and less) reviewing shouldn't be trusted :)

File: go/base/static/css/vumigo.less [hunk 1]
```diff
@@ -6,7 +6,9 @@ body {
   color: #c1c1c1;
 }
 
-.container {
```
Comment (justinvdm) [inline go/base/static/css/vumigo.less:9]: Won't we need to change the things that use the `container` class to use `container-fluid`?

**Comment by smn** [comment]
:+1:

File: go/base/static/css/vumigo.less (hunk 1 above)
Comment (peterp) [inline go/base/static/css/vumigo.less:9]: The old layout used container; but I changed it to container-fluid. You inadvertently made me consider that I should really change bootstrap classes. Thanks!

**Pull request merged**